### DISCUSSION TITLE
[UserStoriesToRefine] #2 I want a website that can easily be translated between languages. The most important in order of priority are Spanish, English, French, and Indonesian

### DISCUSSION THREAD

**Discussion by emecas** [emecas/civic-tech-website-consultancy-pilot-translation#21] [General]
### As <user> I want a website that can easily be translated between languages. The most important in order of priority are Spanish, English, French, and Indonesian

**Comment by nhrafiq** [comment]
## Level 2: As a cacao producer or someone visiting the website, I want to be able to view the website content in a language I best understand so that it is easy to engage with the website content and navigate the website. 
Acceptance criteria:
* Discussion threads #27 and #28 are relevant here. 
* As 28 suggests, the website could be presented to the user in their native location based on their "cookie" location, but could also then prompt the user to pick the language they want to view the website in (as 27 mentions). 
* It should be possible to change the website language from the initial selection anywhere within the website, perhaps by a button that is  available on every page within the website.
